### ISSUE TITLE
Cannot download new subjects to classify -- again

### ISSUE THREAD

**Issue by StevanWhite** [murraycu/android-galaxyzoo#43]
Since late last year, Galaxy Zoo isn't working.

I log in, it thinks a bit, and says
"Cannot download new subjects to classify".
I am able to log in to the Zooniverse web site without problems.
The network is working fine on the phone. I'm on wireless right now.

This is Galaxy Zoo 1.42 (beta) app for my Android Huawei Y350 running Android 4.4.2
See the previous issue #38

**Comment by murraycu** [comment]
Yes, they broke the server API again without warning, but this time it will take me much longer to get it working again, though I am gradually working on it: https://github.com/zooniverse/Galaxy-Zoo/pull/300

**Comment by jeyca** [comment]
what is the state here? is it going to be fixed in the next time? I just downloaded galaxyzoo from fdroid and was disappointed that I couldn't get it to work...

**Comment by murraycu** [comment]
I certainly hope to find time to update the app for the new server API, but my time is limited right now. People are very welcome to submit pull requests. I'm not the only person in the world capable of doing the work.

**Comment by PanderMusubi** [comment]
Hope to see a fix for this, or at least an update width message that it isn't working at the moment. This might prevent losing users.

**Comment by Yky** [comment]
Is there any fix planned for this issue?

I am still experiencing this issue with the F-Droid version (1.64)

**Comment by PanderMusubi** [comment]
This is still a blocking issue. Version 1.64 in the F-Droid app store.
